exatomic/gaussian/output.py

```diff
@@ -608,6 +619,212 @@ def parse_momatrix(self):
         if bcoefs is not None:
             self.momatrix['coef1'] = bcoefs
 
+    def parse_frequency_ext(self):
+        '''
+        Parses extended frequency data present in FChk file
+
+        Note:
+            Requires Freq(SaveNormalModes) in input
+        '''
+        # Frequency regex
+        _renmode = 'Number of Normal Modes'
+        _redisp = 'Vib-Modes'
+        _refinfo = 'Vib-E2'
+        _remass = 'Vib-AtMass'
+        # TODO: find out what these two values are useful for
+        _rendim = 'Vib-NDim'
+        _rendim0 = 'Vib-NDim0'
+
+        found = self.find(_renmode)
+        if not found:
+            return
```

Comment:
There might be a benefit to setting an empty `self.frequency_ext` here prior to exiting. Depends on what the desired behavior is in the case that this parse method fails (doesn't find anything to parse). If the attribute isn't set here, any logic that expects an instance of this class to have the `frequency_ext` attribute will throw an exception. I'm open to ideas/reasons on how to handle this (including maintaining it this way).

Reply:
Sounds like it boils down to downstream code either handling an `AttributeError` on the uni if `frequency_ext` doesn't exist or managing that `frequency_ext` is empty.  Not sure if there is an established precedent but a consistent one is a great idea.

Reply:
So then we'd have to create an empty container for every attribute? Only issue I can see is that if someone is trying to implement some code and is not aware of this they may have a harder time debugging as the error may be linked to their code as opposed to it saying this thing you are using does not exist.